### PR TITLE
0.8.41 monk

### PR DESCRIPTION
**Pull request by wmarkowitz**

### CODE REVIEW

**Comment by wmarkowitz** [comment]
A note about the addition of DamageType to PreDamageTrigger:

I think this will give us more versatility in the future, especially with abilities that perhaps rely on specifically Physical/Magical damage to trigger.

File: cards/src/main/resources/cards/custom/group2/set_633/basic/spell_steadfast_defense.json [hunk 1]
```diff
@@ -64,7 +64,7 @@
       }
     ]
   },
-  "collectible": true,
+  "collectible": false,
```
Comment (doctorpangloss) [inline cards/src/main/resources/cards/custom/group2/set_633/basic/spell_steadfast_defense.json]: this is a token now right?

File: cards/src/main/resources/cards/custom/group2/set_633/core/minion_desciple_of_shitakiri.json [hunk 2]
```diff
@@ -6,7 +6,7 @@
   "baseAttack": 1,
   "baseHp": 7,
   "rarity": "RARE",
-  "description": "When a friendly minion Deflects, gain +2 Attack.",
+  "description": "Whenever a friendly minion Deflects, gain +2 Attack.",
```
Comment (doctorpangloss) [inline cards/src/main/resources/cards/custom/group2/set_633/core/minion_desciple_of_shitakiri.json]: In my experience this card is really strong so just noting it here

File: cards/src/main/resources/cards/custom/group2/set_633/verdant/weapon_verdant_quarterstaff.json [hunk 3]
```diff
@@ -0,0 +1,23 @@
+{
+  "name": "Verdant Quarterstaff",
+  "baseManaCost": 3,
+  "type": "WEAPON",
+  "heroClass": "JADE",
+  "damage": 1,
+  "durability": 6,
+  "rarity": "COMMON",
+  "description": "Extra Strike",
+  "onEquip": {
```
Comment (doctorpangloss) [inline cards/src/main/resources/cards/custom/group2/set_633/verdant/weapon_verdant_quarterstaff.json]: this should probably be done as an `AttributeAura` that does the `AURA_WINDFURY`

File: cards/src/main/resources/cards/custom/group2/set_633/basic/spell_steadfast_defense.json (hunk 1 above)
Comment (wmarkowitz) [reply cards/src/main/resources/cards/custom/group2/set_633/basic/spell_steadfast_defense.json]: this isn't a token, but it was a card that Grand explicitly asked to be made uncollectible for now.

File: cards/src/main/resources/cards/custom/group2/set_633/core/minion_desciple_of_shitakiri.json (hunk 2 above)
Comment (wmarkowitz) [reply cards/src/main/resources/cards/custom/group2/set_633/core/minion_desciple_of_shitakiri.json]: I agree slightly, so I sent a message to Grand about this

Comment (wmarkowitz) [reply cards/src/main/resources/cards/custom/group2/set_633/core/minion_desciple_of_shitakiri.json]: nerfed from 7 health to 6

**Pull request merged**